### ISSUE TITLE
Unable to Create teams app due to login failure

### ISSUE THREAD

**Issue by elafnit** [OfficeDev/microsoft-teams-app-quickstarts#41]
I've been trying to use the Teams Toolkit in vs code. I have installed the latest extension. When I click the toolkit app, "Microsoft Teams", in the vs code ui, I am presented with 2 nav items: Open Microsoft Teams Toolkit and Create a new Teams app. I click the latter and receive a dialog stating "The extension 'Microsoft Teams Toolkit' wants to sign in using Microsoft." I click "Allow" and then am presented with an edge web page with the title "localhost", the url "https://localhost:{randomport}/signin?nonce={random nonce", and the content stating "The connection for this site is not secure. localhost sent an invalid response. ERR_SSL_PROTOCOL_ERROR".
I would very much like to develop Teams apps with this toolkit, but am completely block. Your assistance/direction would be very helpful.

**Comment by asnow003** [comment]
Thank you for the message.  Can you verify that the version of your extension toolkit is at least 1.1.2?  The authentication piece is part of VS Code....I will check around to see what you need to do in order to solve that issue.  What version of Edge are you using?  This is the version I have Version 86.0.622.68 (Official build) (64-bit)

**Comment by elafnit** [comment]
Wow! Very fast response. 😊

I do have toolkit version 1.1.2.
I am using edge Version 86.0.622.63 (Official build) (64-bit)

I currently have my subscription tied to my hotmail account.
My machine is domain joined with my company (not Hotmail).
I develop using my own MS Office dev environment.
I maintain separate edge profiles to minimize account confusions.

My companies Teams is trusted to MS.

Eric Lafnitzegger | Group Manager, IT Software Engineering
ITS Enterprise Applications Services
Avanade Inc.
[ezgif]

**Comment by asnow003** [comment]
I wonder if its because your login might be a hotmail email address.  Is that the email you are trying to login with?  If so...can you use create a user in your dev environment and try logging in with that?

**Comment by elafnit** [comment]
I actually tried that and had the same broken experience

**Comment by asnow003** [comment]
Do you think it would be possible to create an account for me to try on my end.  Email me at allen.snow@microsoft.com with the login and temp password

**Comment by karthigb** [comment]
@elafnit Are you still experiencing this issue?

**Comment by elafnit** [comment]
yes. It would be nice if the extension allowed for --use-device-code. currently, it assumes that it can launch a proper website for the login. I currently have 8 profiles and 5 different logins to separate environments and organizations. This extension just fails for login

**Comment by elafnit** [comment]
I created a VM in azure for development (installed vscode and this extension) to run the samples. Since the original post, I now have a new laptop with less of my accounts associated with vscode and the extension is working as assumed.

**Comment by asnow003** [comment]
That's great to hear... also, maybe since last time VS Code released their "real" login feature which we switched to.  So maybe that helped as well.  Not sure...

thank you for letting us know...

**Comment by i-m-batman** [comment]
Glad that worked for you.

But what I am gathering is that I have to wipe out my computer and start all over to fix this problem? Is there nothing less than erasing VSCode off of the system to get this to work again?

thx
